press/index.html

```diff
@@ -16,11 +16,11 @@ <h4>About NewPipe</h4>
             <p style="margin-bottom: 15px;">A good user experience is important to the developers of NewPipe. Therefore the app offers many solutions for all possible situations. In this way, the resolution and codecs of the video and audio can be set to save data volume in mobile networks. Likewise, NewPipe's video popup and background players make it possible to seamlessly integrate videos and music into daily mobile phone use.</p>
             <p><strong>Quick information:</strong></p>
             <ul>
-                <li>Current stable version: <span class="api-github-stable-version">0.11.5 (Effective 11th February 2018)</span></li>
+                <li>Current stable version: <span data-newpipe-api="flavors.fdroid.stable.version">0.11.5 (Effective 11th February 2018)</span></li>
```

Comment:
Please update to 0.19.5…
NewPipe api uses javascript, therefore if you block it you will see this info. We should not update it every update, but it has been more than two years now.

Plus you did that for contributors and languages below.

Reply:
Done